src/app/modals/new-ig-modal/new-ig-modal.component.ts

```diff
@@ -402,9 +402,9 @@ export class NewIgModal implements OnInit, AfterViewInit {
    * @param group the image group which needs the public property changed
    * @param isPublic the value to set the public property to
```

Comment:
I think we can remove this parameter now?

Reply:
Oh yeah! Good eye!